cc/BUILD

```diff
@@ -10,6 +10,9 @@ DISTRO_DEBS = {
     "debian11": [
```

Comment:
if there are the same, just put them in the tars section. I feel like this might be remnants from debian10 days, and doesn't appear to be necessary anymore.